Makefile

```diff
@@ -330,6 +334,10 @@ test: zdtm
 	$(Q) $(MAKE) -C test
 .PHONY: test
 
+amdgpu_plugin:
+	$(Q) $(MAKE) -C plugins/amdgpu all
```

Comment:
```
$ make amdgpu_plugin
Note: Building criu with amdgpu_plugin.
gcc -g -Wall -Werror -D _GNU_SOURCE -shared -nostartfiles -fPIC amdgpu_plugin.c amdgpu_plugin_topology.c criu-amdgpu.pb-c.c -o amdgpu_plugin.so -iquote../../../criu/include -iquote../../criu/include -lpthread -lrt -ldrm -ldrm_amdgpu -I/usr/include/libdrm
amdgpu_plugin.c: In function ‘amdgpu_plugin_handle_device_vma’:
amdgpu_plugin.c:532:20: error: ‘AMDGPU_KFD_DEVICE’ undeclared (first use in this function)
  532 |         ret = stat(AMDGPU_KFD_DEVICE, &st_kfd);
      |                    ^~~~~~~~~~~~~~~~~
amdgpu_plugin.c:532:20: note: each undeclared identifier is reported only once for each function it appears in
amdgpu_plugin.c: In function ‘amdgpu_plugin_dump_file’:
amdgpu_plugin.c:1273:20: error: ‘AMDGPU_KFD_DEVICE’ undeclared (first use in this function)
 1273 |         ret = stat(AMDGPU_KFD_DEVICE, &st_kfd);
      |                    ^~~~~~~~~~~~~~~~~
amdgpu_plugin.c: In function ‘amdgpu_plugin_restore_file’:
amdgpu_plugin.c:1753:19: error: ‘AMDGPU_KFD_DEVICE’ undeclared (first use in this function)
 1753 |         fd = open(AMDGPU_KFD_DEVICE, O_RDWR | O_CLOEXEC);
      |                   ^~~~~~~~~~~~~~~~~
amdgpu_plugin.c: In function ‘amdgpu_plugin_update_vmamap’:
amdgpu_plugin.c:2176:27: error: ‘AMDGPU_KFD_DEVICE’ undeclared (first use in this function)
 2176 |         if (!strcmp(path, AMDGPU_KFD_DEVICE))
      |                           ^~~~~~~~~~~~~~~~~
amdgpu_plugin.c: In function ‘amdgpu_plugin_resume_devices_late’:
amdgpu_plugin.c:2211:19: error: ‘AMDGPU_KFD_DEVICE’ undeclared (first use in this function)
 2211 |         fd = open(AMDGPU_KFD_DEVICE, O_RDWR | O_CLOEXEC);
      |                   ^~~~~~~~~~~~~~~~~
make[1]: *** [Makefile:27: amdgpu_plugin.so] Error 1
```

Reply:
ACK, previous force push had one change that modified criu-plugin.h and introduced this error. Fixed now with next force-push